src/api/blockchain.js

```diff
@@ -27,6 +27,18 @@ const Blockchain = {
     return qClient.getBlockCount();
   },
 
+  async getBlockHash(args) {
```

Comment:
you might already know this, but you can destructure this in the props.
i.e.
```
async getBlockHash({ blockNum }) {
  ...
}
```

Reply:
true, just wanted to follow the same structure all the other functions for now. will do a refactor later to change all of them at once.